packages/projection-typeorm/src/operators/storePoolMetricsUpdateJob.ts

```diff
@@ -3,6 +3,9 @@ import { STAKE_POOL_METRICS_UPDATE } from '../pgBoss';
 import { WithPgBoss } from './withTypeormTransaction';
 import { typeormOperator } from './util';
 
+export const willStorePoolMetricsUpdate = ({ eventType }: { eventType: ChainSyncEventType }) =>
+  eventType !== ChainSyncEventType.RollBackward;
```

Comment:
We don't have to run this for all blocks, we can either
- use the same logic as in the store operator: once reached the tip and every `jobFrequency` blocks
- always return `false` and update the store operator logic to insert the job on 1st suitable block we see, e.g. if last seen block is 999, jobFrequency=1000, and we see block number 1111 then we would insert the job.

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1200/commits/36f6b38e783c9ac91eea234884c3153013f6f018

Reply:
Please amend the commit message to mention this change of `storeStakePoolMetricsUpdateJob` behavior

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1200/commits/307254250a1c35ea1fcab16287218672aacab9e8